schema.json

```diff
@@ -1462,6 +1462,587 @@
     ]
   },
   {
+    "name": "Optimized Checkout",

```

Comment:
I wonder if we should hide this tab if you are not using UCO. Otherwise it will be confusing when users try to change a setting and nothing changes.

We could add an attribute that shows/hide depending on a store configuration. Something like this:

```json
{
    "name": "Optimized Checkout",
    "enabled": "ucoEnabled",
    "settings": [ 
        ...
    ]
}
```